scripts/upgrade

```diff
@@ -91,6 +96,23 @@ ynh_script_progression --message="Stopping a systemd service..."
 
 ynh_systemd_action --service_name=$app --action="stop" --log_path=systemd --line_match="Stopped"
 
+#=================================================
+# UPGRADE FROM PREVIOUS VERSION
+#=================================================
+
+if ynh_version_gt "1.0.0-rc1~ynh1" "${previous_version}" ; then
```

Comment:
eeeeh are we sure about the comparisons like `1.0.0-rc1~ynh1` vs `1.0.0-rc.4~ynh1` ? (note the dot after rc ?)

Reply:
yes, as 1.0.0-rc1~ynh1 has nether gone  to master, we can directly go to 1.0.0